coderd/roles_test.go

```diff
@@ -120,75 +120,96 @@ func TestListRoles(t *testing.T) {
 	require.NoError(t, err, "create org")
 
 	const forbidden = "Forbidden"
-	siteRoles := convertRoles(rbac.RoleOwner(), "auditor", "template-admin", "user-admin")
-	orgRoles := convertRoles(rbac.RoleOrgAdmin(admin.OrganizationID))
+	//siteRoles := convertRoles(rbac.RoleOwner(), "auditor", "template-admin", "user-admin")
+	//orgRoles := convertRoles(rbac.RoleOrgAdmin(admin.OrganizationID))
```

Comment:
Remove commented code or add a comment explaining why it's commented out (is it an example?)